examples/plot_simulate_alpha.py

```diff
@@ -19,47 +23,87 @@
 from hnn_core import simulate_dipole, read_params, Network
 
 ###############################################################################
-# Then we setup the directories and Neuron
+# Then we setup the directories and read the default parameters file
 hnn_core_root = op.dirname(hnn_core.__file__)
-
-###############################################################################
-# Then we read the default parameters file
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 params = read_params(params_fname)
 print(params)
 
-###############################################################################
-# Update a few of the default parameters related to visualisation
-params.update({
-    'dipole_scalefctr': 150000.0,
-    'dipole_smooth_win': 0,
-    'tstop': 310.0,
-})
-
 ###############################################################################
 # Now let's simulate the dipole and plot it. To excite the network, we add a
 # ~10 Hz "bursty" drive starting at 50 ms and continuing to the end of the
 # simulation. Each burst consists of a pair (2) of spikes, spaced 10 ms apart.
 # The occurrence of each burst is jittered by a random, normally distributed
 # amount (20 ms standard deviation). We repeat the burst train 10 times, each
-# time with unique randomization. The drive is only connected to the distal
+# time with unique randomization. The drive is only connected to the proximal
 # (dendritic) AMPA synapses on L2/3 and L5 pyramidal neurons.
+params['tstop'] = 310
 net = Network(params)
 
-weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+location = 'proximal'
+burst_std = 20
+weights_ampa_p = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+syn_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                'L5_basket': 1., 'L5_pyramidal': 1.}
+
 net.add_bursty_drive(
-    'bursty', tstart=50., burst_rate=10, burst_std=20., numspikes=2,
-    spike_isi=10, repeats=10, location='distal', weights_ampa=weights_ampa,
-    seedcore=4)
+    'alpha_prox', tstart=50., burst_rate=10, burst_std=burst_std, numspikes=2,
+    spike_isi=10, repeats=10, location=location, weights_ampa=weights_ampa_p,
+    synaptic_delays=syn_delays_p, seedcore=14)
 
-dpl = simulate_dipole(net)
+dpl = simulate_dipole(net, postproc=False)
 
+###############################################################################
+# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
+# power spectral density. Note that the SciPy-function
+# `~scipy.signal.spectrogram` is used to create the plot. The
+# ``dpl[trial_idx].scale()`` call relates to the amount of cortical tissue
+# necessary to observe the electric current dipole outside the head with M/EEG.
+import matplotlib.pyplot as plt
+from hnn_core.viz import plot_dipole, plot_spectrogram
 trial_idx = 0  # single trial simulated
-dpl[trial_idx].plot()
+dpl[trial_idx].scale(150000)
```

Comment:
This is a bit weird: the user will wonder what this magic number is about? I vote for just removing it, since the present example is for visualisation only.

Reply:
If we remove this, I'm not sure the dipoles will be in nAm any more? We need to dig into this a bit ... folks who are sticklers about units may complain

Reply:
In any case, I agree that there needs to be an explanation. Something like: "We can scale the signals by xxxx which implies that xxx neurons are needed to produce an alpha wave of amplitude yyy nAM"

Reply:
Based on `Dipole.convert_fAm_to_nAm`, the default scale is fAm. So `1.5e5` doesn't match anything, I think. Note that I purposefully left out the units in a recent commit to `viz.plot_dipole`: _all this should be checked!_

Arbitrary scaling to 'model' larger populations should be covered in a separate example, IMHO. Here we're just interested in the waveform, not worried about the units.

Reply:
Consensus seems to be chucking the scaling so I am removing. I agree though this is a technical detail worth addressing separately.